README.md

```diff
@@ -31,12 +30,40 @@ fastify.register(oauthPlugin, {
 })
 
 fastify.get('/login/facebook/callback', async function (request, reply) {
-  const result = await this.getAccessTokenFromAuthorizationCodeFlow(request)
+  const result = await this.facebookOAuth2.getAccessTokenFromAuthorizationCodeFlow(request)
 
   console.log(result.access_token)
 
   reply.send({ access_token: result.access_token })
 })
+
+fastify.register(oauthPlugin, {
+	name: 'githubOAuth2',
+	credentials: {
+	  client: {
+	    id: 'my-client-id',
+	    secret: 'my-secret'
+    },
+	  auth: oauthPlugin.GITHUB_CONFIGURATION
+	},
+	startRedirectPath: '/login/github',
+	callbackUri: 'http://localhost:3000/login/github/callback',
+	scope: ['notifications']
+})
+
+fastify.get('/login/github/callback', function (request, reply) {
+	return this.githubOAuth2.getAccessTokenFromAuthorizationCodeFlow(request)
+	  .then(result => {
+	    const token = this.githubOAuth2.accessToken.create(result)
+	    return {
+        access_token: token.token.access_token,
+	      refresh_token: token.token.refresh_token,
+	      expires_in: token.token.expires_in,
+	      token_type: token.token.token_type
+	    }
+	  })
```

Comment:
Can you please lint example? There seems a bit too many spaces at the beginning of the line.